androidtv/constants.py

```diff
@@ -269,41 +269,65 @@
 
 # Apps
 APP_AMAZON_VIDEO = 'com.amazon.avod'
+APP_AMZ_VIDEO = 'com.amazon.avod.thirdpartyclient'
```

Comment:
This variable name doesn't really matter because it's only used internally, but how about `APP_AMAZON_VIDEO_3RD_PARTY` or `APP_AMAZON_VIDEO_THIRD_PARTY`?